### ISSUE TITLE
Password validation unmasks text

### ISSUE THREAD

**Issue by bialesdaniel** [SBoudrias/Inquirer.js#553]
I have a question configured like so:
```
{
  name: 'password',
  type: 'password',
  message: 'Password?',
  validate: (input) => {
    return (input.length === 8) ? true : 'Input your 8 char password'
  }
}
```
If the user enters a bad password then the next time they type it, it will appear in clear text in the terminal.

**Comment by marcielmj** [comment]
Having the same issue here.

**Comment by ngotchac** [comment]
This should fix it : #546
